doc/about/about.rst

```diff
@@ -1,72 +1,108 @@
-Frequently Asked Questions
```

Comment:
If we move the page we need a redirect from the old URL to the new one.

Reply:
if we move all of these docs to binderhub does that mean we need to set up a redirect for every old doc page to every new doc page? Do we know how many direct links to binder docs there are? I suspect not that many...